_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,118 @@
+
+ A good CMS must provide means for your marketing team to work independently from your development team. Drupal 8 does just that! It provides solid framework that can be tuned to become a powerful marketing-machine.
+
+ Let’s take a look at some of the new feature in core that make search engines love Drupal 8.
+
+#### Drupal 8 is Responsive out of the Box
+
+Drupal 8 comes with responsive themes in core. Now both public facing and admin facing themes are responsive and make user experience great on any device.
+
+#### Drupal 8 Page Load is Fast
+
+No! Not out the box! In fact vanilla Drupal 8 is running much more code than vanilla Drupal 7. It runs vendor code like Symfony, which adds some overhead. However, Drupal 8 has a number of improvements that are making up for that overhead:
+
+- **Javascript files are now loading in the footer**. Due to this change pages build up faster and user can see and use them earlier.
+
+- **[Pluggable CSS/JS aggregation and minification](https://www.drupal.org/node/2034675)** to support more optimal optimization algorithms
+
+- **Highly improved caching**. Drupal 8 uses “cache tags” that makes caching more efficient and includes Cache Context API which provides context-based caching. This means pages load faster while ensuring that visitors always see the latest version of your site.
+
+- **BigPipe render pipeline**. Sends pages in a way that allows browsers to show them much faster. First sends the cacheable parts of the page, then the dynamic/uncacheable parts. Uses the [BigPipe technique](https://www.facebook.com/notes/facebook-engineering/bigpipe-pipelining-web-pages-for-high-performance/389414033919).
+
+These improvements have the potential to make your Drupal 8 website fly! And if after all that it is not “flying” - than you need someone to review the code that powers your website’s features. ([Contact us](/contact/))
+
+#### Semantic Markup
+
+Search engines appreciate clean markup that explicitly describes the purpose of on-page elements. Thanks to [HTML5 Initiative](https://www.drupal.org/community-initiatives/drupal-core/html5) for Drupal 8 development, we now have a number of great markup improvements right in Drupal core:
+
+- HTML5 themes with new semantic elements in core templates
+
+- Support for the new form elements to Drupal's Form API
+
+- Rich media handling with \<video> and \<audio> elements
+
+- ARIA roles in markup to improve accessibility
+
+- Twig theming engine - makes it harder for developers to create messy, non-semantic code
+
+#### Content as a Service
+
+Another exciting new feature of Drupal 8 is a flexible content delivery.
```

Comment:
This area in general is a little vague. I think clients will have a hard time understanding what you're getting at here. Can you give a bit more context, or speak about the benefit in a way that's clear to the non-technical?